dev/PushNotifications/PushNotificationsLongRunningTask/platform.h

```diff
@@ -18,18 +15,28 @@ struct __declspec(uuid(PUSHNOTIFICATIONS_IMPL_CLSID_STRING)) NotificationsLongRu
 
     STDMETHOD(RegisterFullTrustApplication)(_In_ PCWSTR processName, _In_ GUID remoteId, _Out_ GUID* appId) noexcept;
 
+    STDMETHOD(RegisterActivator)(_In_ PCWSTR processName) noexcept;
+
+    STDMETHOD(UnregisterActivator)(_In_ PCWSTR processName) noexcept;
+
     STDMETHOD(RegisterForegroundActivator)(_In_ IWpnForegroundSink* sink, _In_ PCWSTR processName);
```

Comment:
noexcept for these 2 as well

Reply:
I should be more attentive to such details.